CFC_DataCollector/trip_generator/input.json

```diff
@@ -1,15 +0,0 @@
-{
-	"radius" : "1",
```

Comment:
Why delete this? If you delete it, people will have to reverse engineer the format by looking at the code, a process that which 99% of the people in the world hate. Having the input file here helps a lot. If you are concerned about privacy or inadvertent use. use dummy values and rename it to input.json.sample if you like.